src/main/java/gov/cms/madie/models/measure/PopulationType.java

```diff
@@ -19,6 +19,12 @@ public enum PopulationType {
   DENOMINATOR_EXCEPTION("denominator-exception","Denominator Exception"),
   @JsonProperty("measurePopulation")
   MEASURE_POPULATION("measure-population","Measure Population"),
+  @JsonProperty("measurePopulationObservation")
+  MEASURE_POPULATION_OBSERVATION("measure-population-observation","Measure Population Observation"),
```

Comment:
I think the code and display for all three measure observation types should just be "measure-observation" and "Measure Observation" cause these were tied to the FHIR types